docs/root/configuration/listeners/network_filters/connection_limit_filter.rst

```diff
@@ -0,0 +1,106 @@
+.. _config_network_filters_connection_limit:
+
+Connection Limit Filter
+=======================
+
+Background
+----------
+
+Network connections are a limited resource that we need some functionality to protect.
```

Comment:
See my other comment about the listener connection limits. We should make sure to reference both features here and in the general connection limiting overview.

Reply:
Got it, updated.